src/cursor/cur_backup.c

```diff
@@ -890,7 +896,8 @@ __backup_list_uri_append(WT_SESSION_IMPL *session, const char *name, bool *skip)
      */
     if (!WT_PREFIX_MATCH(name, "file:") && !WT_PREFIX_MATCH(name, "colgroup:") &&
       !WT_PREFIX_MATCH(name, "index:") && !WT_PREFIX_MATCH(name, "lsm:") &&
-      !WT_PREFIX_MATCH(name, WT_SYSTEM_PREFIX) && !WT_PREFIX_MATCH(name, "table:") &&
+      !WT_PREFIX_MATCH(name, "object:") && !WT_PREFIX_MATCH(name, WT_SYSTEM_PREFIX) &&
```

Comment:
Do we expect the details of `object:` and `tier:` types of files to be in the WiredTiger.export file? I noticed these were previously not in the list of expected URIs and the Python test fails without adding them in.

Reply:
My understanding is that all the details should be in the `WiredTiger.export` to allow the import procedure to add them into the recipient's metadata.

Reply:
Yes, we would. That is possibly a bug in the POC code that it is not included.